kerl

```diff
@@ -932,8 +932,23 @@ ACTIVATE_CSH
         build_plt "$absdir"
     fi
 
+    PID=$$
+    PPID=$(ps -p $PID -o ppid | grep -E "[0-9]+" -o)
```

Comment:
Clever I like this. Just wonder if it's portable across Mac and Linux.  Might need to stuff those flags up with the SED_OPTS and MD5_OPTS above if TravisCI complains.